cumulus_library/cli.py

```diff
@@ -291,7 +291,7 @@ def run_cli(args: dict):
     if args["action"] == "upload":
         try:
             uploader.upload_files(args)
-        except requests.RequestException as e:
+        except requests.RequestException as e:  # pragma: no cover
```

Comment:
Why not test this path? An error during upload feels like a reasonable test to have.

Reply:
you found my 'why did this break' frustration! i'll figure it out